lsp/server/src/test/TestHelper.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
```

Comment:
Copyright year

Reply:
@sfdctaka really impressed by your meticulous attentions to detail.